lib/Console/Command/RunCommand.php

```diff
@@ -40,16 +41,38 @@ public function configure()
         );
         $this->addArgument('path', InputArgument::OPTIONAL, 'Path to benchmark(s)');
         $this->addOption('report', array(), InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Report name or configuration in JSON format');
-        $this->addOption('filter', null, InputOption::VALUE_REQUIRED, 'Filter subject(s) to run');
-        $this->addOption('dumpfile', 'df', InputOption::VALUE_REQUIRED, 'Dump XML to named file');
+        $this->addOption('subject', null, InputOption::VALUE_REQUIRED, 'Subject to run');
+        $this->addOption('dumpfile', 'df', InputOption::VALUE_OPTIONAL, 'Dump XML result to named file');
+        $this->addOption('dump', null, InputOption::VALUE_NONE, 'Dump XML result to stdout');
+        $this->addOption('parameters', null, InputOption::VALUE_REQUIRED, 'Explicit parameters to use in benchmark');
+        $this->addOption('nosetup', null, InputOption::VALUE_NONE, 'Do not execute setUp or tearDown methods');
+        $this->addOption('separateprocess', null, InputOption::VALUE_NONE, 'Run iterations in separate processes');
```

Comment:
Should be changed to enum `run|iterations`. i.e. run each iteration SET in a separate process or every individual iteration (a RUN).